awx/main/notifications/hipchat_backend.py

```diff
@@ -26,10 +26,13 @@ class HipChatBackend(AWXBaseEmailBackend):
     def __init__(self, token, color, api_url, notify, fail_silently=False, **kwargs):
         super(HipChatBackend, self).__init__(fail_silently=fail_silently)
         self.token = token
-        self.color = color
+	if color == 'None':
```

Comment:
Well, this isn't really what I meant... I don't much care if it's a literal string value of `"None"` so much as I want to make sure we don't call `color.lower()` when `color` is the literal Python `None`

Which would lead to the following exception:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'NoneType' object has no attribute 'lower'
```